openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/ActivityFeedTab.component.tsx

```diff
@@ -109,6 +113,33 @@ export const ActivityFeedTab = ({
   const [taskFilter, setTaskFilter] = useState<ThreadTaskStatus>(
     ThreadTaskStatus.Open
   );
+  const [announcementFilter, setAnnouncementFilter] =
+    useState<AnnoucementStatus>(AnnoucementStatus.Active);
+  const [isAddAnnouncementOpen, setIsAddAnnouncementOpen] =
+    useState<boolean>(false);
+  const handleCloseAnnouncementModal = useCallback(
+    () => setIsAddAnnouncementOpen(false),
+    []
+  );
+  const handleOpenAnnouncementModal = useCallback(
+    () => setIsAddAnnouncementOpen(true),
+    []
+  );
+
+  const handleSaveAnnouncement = useCallback(() => {
+    handleCloseAnnouncementModal();
+    getFeedData(
+      feedFilter,
+      undefined,
+      ThreadType.Announcement,
+      entityType,
+      fqn,
+      undefined,
+      undefined,
+      announcementFilter
+    );
+  }, [announcementFilter]);
```

Comment:
I believe we should use the `handleCloseAnnouncementModal` after fetching the feedData. And can you also check the dependency, there are some missing value that should be added!

Reply:
done, updated dependencies for this and  made  handleCloseAnnouncementModal  function call after fetching feeddata